libgalois/src/analytics/bfs/bfs.cpp

```diff
@@ -52,11 +52,14 @@ constexpr bool kTrackWork = BfsImplementation::kTrackWork;
 
 using UpdateRequest = BfsImplementation::UpdateRequest;
 using Dist = BfsImplementation::Dist;
-using SrcEdgeTile = BfsImplementation::SrcEdgeTile;
-using SrcEdgeTilePushWrap = BfsImplementation::SrcEdgeTilePushWrap;
 using ReqPushWrap = BfsImplementation::ReqPushWrap;
 using OutEdgeRangeFn = BfsImplementation::OutEdgeRangeFn;
-using TileRangeFn = BfsImplementation::TileRangeFn;
+
+struct BfsNodeDataTy {
+  uint32_t parent;
+  Dist distance;
+};
+using BfsNodeData = struct BfsNodeDataTy;
```

Comment:
Please try a structure of arrays as well. It might be faster. Please confirm that before you merge.